stanford-sw/translation_maps/library_map.properties

```diff
@@ -36,3 +36,5 @@ SPEC-COLL = Special Collections
 TANNER = Philosophy (Tanner)
 
 #SUL = Stanford University Libraries
+MEDIA-MTXT = MEDIA-MTXT
+RUMSEYMAP = RUMSEYMAP

```

Comment:
I would add a comment as to why they are at the bottom with non-user friendly strings, but not a deal breaker.

I'm not sure if you *need* them in the translation_map b/c the default *may* be to display the raw value but you might as well leave it as is since it's there and it works (assuming travis passes).  

No doubt you are also intending to stick them in alphabetically (or not) once they are official from JV with approved strings?